eq3bt/eq3btsmart.py

```diff
@@ -140,6 +147,20 @@ def handle_notification(self, data):
             else:
                 self._mode = Mode.Auto
 
+            presets = status.presets
```

Comment:
Do you know if this can/will break on older firmware versions?

Reply:
This part specifically will not affect functionality on older firmware versions, but my changes on ``Status`` in the ``structures`` module could.
I implemented the tests before changing this, and later added an additional response version with the presets; but since I have never actually seen the output of a thermostat with an older firmware revision, I could only make assumptions of what the old status responses should be like, based on the prior ``Status`` and other implementations I searched for.

I would therefore be quite surprised if this broke anything, but I cannot be absolutely sure that it won't. It would be great if someone could test this, since I have no idea how to downgrade a thermostat, or if that is even possible.

Reply:
Considering that there has been no rush of volunteers, we can simply merge this and prepare fixes if necessary. Could you please rebase this on the current master?

Reply:
I created a new PR #33 with rebased changes.